src/mame/philips/minitel_2_rpic.cpp

```diff
@@ -564,23 +576,28 @@ void minitel_state::minitel2(machine_config &config)
 }
 
 ROM_START( minitel2 )
-
 	ROM_REGION( 0x10000, "maincpu", ROMREGION_ERASEFF )
 	ROM_DEFAULT_BIOS("ft_bv4")
 
-	ROM_SYSTEM_BIOS(0, "ft_bv4", "Minitel 2 ROM BV4")
+	ROM_SYSTEM_BIOS(0, "ft_bv4", "France Telecom Bv4")
 	ROMX_LOAD( "minitel2_bv4.bin",   0x0000, 0x8000, CRC(8844a0a7) SHA1(d3e9079b080dbcee27ad870ec6c39ac42e7deacf), ROM_BIOS(0) )
 
-	ROM_SYSTEM_BIOS(1, "demov1", "Minitel 2 Demo")
-	ROMX_LOAD( "demo_minitel.bin",   0x0000, 0x8000, CRC(607f2482) SHA1(7965edbef68e45d09dc67a4684da56003eff6328), ROM_BIOS(1) )
+	ROM_SYSTEM_BIOS(1, "ft_bv6", "France Telecom Bv6")
+	ROMX_LOAD( "bv6.bin",            0x0000, 0x8000, CRC(9e162977) SHA1(e66a0951d14be4f9da693c478319d1c620878347), ROM_BIOS(1) )
+
+	ROM_SYSTEM_BIOS(2, "ft_bv7", "France Telecom Bv7")
+	ROMX_LOAD( "bv7.bin",            0x0000, 0x8000, CRC(2dde1628) SHA1(85500c06b93efb6e925dcb156248425fa3366ce6), ROM_BIOS(2) )
+
+	ROM_SYSTEM_BIOS(3, "ft_bv9", "France Telecom Bv9")
+	ROMX_LOAD( "bv9.1402",           0x0000, 0x8000, CRC(ace5d65e) SHA1(c8d589f8af6bd7d339964fdece937a76db972115), ROM_BIOS(3) )
 
-	ROM_SYSTEM_BIOS(2, "ft_bv9", "Minitel 2 ROM Bv9")
-	ROMX_LOAD( "bv9.1402",           0x0000, 0x8000, CRC(ace5d65e) SHA1(c8d589f8af6bd7d339964fdece937a76db972115), ROM_BIOS(2) )
+	ROM_SYSTEM_BIOS(4, "demov1", "The Minitel Demo (jfdelnero)")
+	ROMX_LOAD( "demo_minitel.bin",   0x0000, 0x8000, CRC(607f2482) SHA1(7965edbef68e45d09dc67a4684da56003eff6328), ROM_BIOS(4) )
```

Comment:
Stuff like this should be in a software list (if it should be in MAME itself at all).